test/cqlpy/test_ssl.py

```diff
@@ -37,15 +53,31 @@ def test_tls_versions(cql):
                         ssl.TLSVersion.TLSv1,
                         ssl.TLSVersion.SSLv3]:
         try:
-            try_connect(cql.cluster, ssl_version)
-            print(f"{ssl_version} supported")
+            with try_connect(cql.cluster, ssl_version) as session:
+                print(f"{ssl_version} supported")
+                table_result = session.execute(f"SELECT * FROM system.clients")
+                for row in table_result:
+                    assert row.ssl_enabled == True     # ' == True` is redundant, but makes it clear
```

Comment:
Please remove the comment and/or the "== True".
If you added "== True" to "make it clear", but then needed a comment to "make it clear you made it clear", something is wrong.
I think either
```
assert row.ssl_enabled == True
```
or
```
assert row.ssl_enabled
```
is good and don't need a comment. The latter assumes that ssl_enabled has the boolean type (so it only be a None or a boolean but nothing else) but this assumption is true so it's fine.